code/modules/hydroponics/plants_flower.dm

```diff
@@ -20,3 +20,37 @@ ABSTRACT_TYPE(/datum/plant/flower)
 	force_seed_on_harvest = 1
 	mutations = list()
 	commuts = list(/datum/plant_gene_strain/immunity_radiation,/datum/plant_gene_strain/damage_res/bad)
+
+/datum/plant/flower/rafflesia
+	name = "Rafflesia"
+	seedcolor = "#A4000F"
+	crop = /obj/item/clothing/head/rafflesia
+	growthmode = "weed"
+	starthealth = 40
+	growtime = 50
+	harvtime = 90
+	cropsize = 1
+	harvests = 1
+	endurance = 5
+	genome = 9
+	force_seed_on_harvest = 1
+	special_proc = 1
+	mutations = list()
+	commuts = list(/datum/plant_gene_strain/resistance_drought)
+	assoc_reagents = list("miasma")
+	
+	HYPspecial_proc(var/obj/machinery/plantpot/POT) // Smokes miasma and whatever chemicals have been spliced into the plant
+		..()
+		if (.) return
+		var/datum/plant/P = POT.current
+		var/datum/plantgenes/DNA = POT.plantgenes	
+		var/spray_prob = max(33,(33 + DNA.endurance / 5))
+		var/datum/reagents/reagents_temp = new/datum/reagents(max(1,(50 + DNA.cropsize))) // Creating a temporary chem holder
+		reagents_temp.my_atom = POT
+	
+		if (POT.growth > (P.harvtime - DNA.growtime) && prob(spray_prob))
+			for(var/num in range(round(max(1,(1 + DNA.potency / 10)))))
+				for(var/REAG in assoc_reagents)
+					reagents_temp.add_reagent(REAG, 3)
```

Comment:
```suggestion
			reagents_temp.add_reagent("miasma", 3 * round(max(1,(1 + DNA.potency / 10))))
```

Reply:
Again, I don't see this necessary at all. You can't splice this plant with anything yet anyway, thanks to the HYPspecial not working in any spliced plant. This is just useless botany hate.

Reply:
Yeah, but this is inefficent coding otherwise. I actually though I had done this exact change, but i have like 4 repositories when i was playing around with it a bit more so i must have done it in one of those.